### PR TITLE
PIM-5602: Add "move to categories" action to mass edit

### PR DESCRIPTION
**Pull request by phaseinducer**
**Description (for Contributor and Core Developer)**

This PR:
- Moves products mass action buttons into a dropdown.
- Splits mass edit operations into two groups: "Mass Edit" And "Category Edit". Both pages use the same templates, same actions, the groups are simply defined in the DI tags on operations.
- Adds the "move to categories" operation (alias `classify-move`).
- Renames the operation with alias `classify` to `classify-add`.
- Simplify mass edit behats by removing some useless step definitions (imho).

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | yes
| Added Behats                      | yes
| Changelog updated                 | yes
| Review and 2 GTM                  |
| Micro Demo to the PO (Story only) |
| Migration script                  | n/a
| Tech Doc                          |

### CODE REVIEW

File: features/mass-action/change_family.feature [hunk 1]
```diff
@@ -28,7 +29,8 @@ Feature: Change family of many products at once
     And the family of product "hamburger" should be "Food"
 
   Scenario: Remove many products from a product family
-    Given I mass-edit products coffee, hamburger and jeans
+    Given I select rows coffee, hamburger and jeans
+    And I press "Mass Edit" on the "Bulk Actions" dropdown button
```
Comment (juliensnz) [inline features/mass-action/change_family.feature]: so much love

File: features/mass-action/change_status.feature [hunk 2]
```diff
@@ -23,7 +24,8 @@ Feature: Configure action to change status of many products at once
     Given an enabled "boat" product
     And an enabled "jet-ski" product
     And I am on the products page
-    When I mass-edit products boat and jet-ski
+    When I select rows boat and jet-ski
+    And I press "Mass Edit" on the "Bulk Actions" dropdown button
```
Comment (juliensnz) [inline features/mass-action/change_status.feature]: raaah this is so good

File: features/mass-action/edit_common_attributes-01.feature [hunk 3]
```diff
@@ -31,7 +31,8 @@ Feature: Edit common attributes of many products at once
     And I am on the products page
 
   Scenario: Allow editing all attributes on configuration screen
-    Given I mass-edit products boots, sandals and sneakers
+    Given I select rows boots, sandals and sneakers
+    And I press "Mass Edit" on the "Bulk Actions" dropdown button
```
Comment (juliensnz) [inline features/mass-action/edit_common_attributes-01.feature]: I can't love it more

File: features/mass-action/edit_common_attributes-01.feature [hunk 4]
```diff
@@ -52,7 +54,8 @@ Feature: Edit common attributes of many products at once
     And the english name of "sneakers" should be "boots"
 
   Scenario: Successfully update many multi-valued values at once
-    Given I mass-edit products boots and sneakers
+    Given I select rows boots and sneakers
+    And I press "Mass Edit" on the "Bulk Actions" dropdown button
```
Comment (juliensnz) [inline features/mass-action/edit_common_attributes-01.feature]: Did I already said that I love this modification?

File: features/mass-action/edit_common_attributes-01.feature [hunk 5]
```diff
@@ -101,7 +106,8 @@ Feature: Edit common attributes of many products at once
     Given the following product values:
       | product | attribute          | value   |
       | boots   | weather_conditions | dry,hot |
-    Given I mass-edit products boots and sneakers
+    Given I select rows boots and sneakers
+    And I press "Mass Edit" on the "Bulk Actions" dropdown button
```
Comment (juliensnz) [inline features/mass-action/edit_common_attributes-01.feature]: :honeybee:

File: src/Pim/Bundle/EnrichBundle/MassEditAction/Operation/OperationRegistry.php [hunk 6]
```diff
@@ -45,8 +45,13 @@ public function __construct(
      *
      * @throws \InvalidArgumentException If the operation is already registered
      */
-    public function register(MassEditOperationInterface $operation, $operationAlias, $acl = null, $gridName = null)
-    {
+    public function register(
+        MassEditOperationInterface $operation,
+        $operationAlias,
+        $gridName,
```
Comment (phaseinducer) [inline src/Pim/Bundle/EnrichBundle/MassEditAction/Operation/OperationRegistry.php]: BC break

**Comment by phaseinducer** [comment]
Mob reviewed with @pierallard, @grena and @skeleton.

**Pull request merged**